.github/workflows/maven.yml

```diff
@@ -27,7 +28,8 @@ jobs:
         key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
         restore-keys: |
           ${{ runner.os }}-maven-
-    - name: Build with Maven
-      run: ./mvnw -V -B -ntp -ff install '-DskipTests=true' '-Dmaven.javadoc.skip=true'
     - name: Run tests
-      run: ./mvnw -V -B -ntp -ff verify
+      run: ./mvnw -V -B -ntp -ff clean verify
```

Comment:
Since we're running from scratch every time the workflow is being executed, the `clean` phase can be skipped.